src/layouts/Docs.tsx

```diff
@@ -27,47 +17,33 @@ import FeedbackCard from "@/components/FeedbackCard"
 import FileContributors from "@/components/FileContributors"
 import GlossaryTooltip from "@/components/Glossary/GlossaryTooltip"
 import InfoBanner from "@/components/InfoBanner"
-import Link from "@/components/Link"
 import MainArticle from "@/components/MainArticle"
 import {
   Heading1 as MdHeading1,
   Heading2 as MdHeading2,
   Heading3 as MdHeading3,
   Heading4 as MdHeading4,
-  Paragraph,
 } from "@/components/MdComponents"
 import RollupProductDevDoc from "@/components/RollupProductDevDoc"
 import SideNav from "@/components/SideNav"
 import SideNavMobile from "@/components/SideNavMobile"
 import TableOfContents from "@/components/TableOfContents"
 import Translation from "@/components/Translation"
 import { Divider } from "@/components/ui/divider"
+import InlineLink from "@/components/ui/Link"
 import { mdxTableComponents } from "@/components/ui/Table"
 import YouTube from "@/components/YouTube"
 
 import { cn } from "@/lib/utils/cn"
 import { getEditPath } from "@/lib/utils/editPath"
 
-const Page = (props: ChildOnlyProp & Pick<FlexProps, "dir">) => (
-  <Flex
-    direction="column"
-    w="full"
-    borderBottom="1px"
-    borderColor="border"
-    {...props}
-  />
+const Page = (props: HTMLAttributes<HTMLDivElement>) => (
+  <div className="flex w-full flex-col border-b" {...props} />
```

Comment:
@pettinarip Do we care to clean up some of these one-off named components? `Page`, `ContentContainer`, and `Content` are all only used once bellow in the return, and not exported... Nothing critical, but would be nice imo to clean these up.

Reply:
Good idea! Done